README.md

```diff
@@ -1,3 +1,75 @@
 # REACT EXPRESS GRAPHQL TEMPLATE
 
-This template is meant to provide the user with the barebones boilerplate for a React/Express project built with a GraphQL API and a PostgreSQL database.
\ No newline at end of file
+This template is meant to provide the user with the barebones boilerplate for a React/Express project built with a GraphQL API and a PostgreSQL database.
+
+The use of the this template repo means that you agree to abide by the current licensing rules.
+
+##### NOTE: This template is still under development.
+
+## Template use-cases
+
+This template is intended to be used as a very quick project setup. For that reason, it only supports a single software stack. This stack includes:
+
+### Front-end
+
+- react
+- react-router-dom
+- apollo
+
+### Back-end
+
+- node / express
+- REST
+- graphql
+- massive (postgresql)
+- db-migrate
+
+### Developer Experience
+
+- nodemon
+- winston
+- chalk
+- dotenv
+- eslint
+- prettier
+- husky
+- jest
+- babel
+
+Each of these packages can be found on npmjs.org for more information.
+
+## Setup Instructions
+
+This template is intended to be an out-of-the-box way to set up a new project in a matter of minutes. Once a repo has been created off of this template, the user can clone the new repo to their preferred machine, run `npm install`, change the needed variables, and then run `npm start` and `npm run start:server` to get the project running!
+
+### Create new repo
+
+Do not use this template directly. You must create a new repo on your own github account using this template (this is all done on the github.com website).
+
+Once you have created a new repo off of this template, clone and download it to your computer.
+
```

Comment:
maybe link to the directions for how to do that?

Reply:
added in PR update